isis/src/base/apps/caminfo/caminfo.xml

```diff
@@ -20,191 +20,189 @@
     running the spiceinit program from within caminfo.
   </p>
   <p>
-    Some information in the output PVL file will consist of a compilation of information for all the 
-    bands in the cube and will be listed in the "Common" object of the PVL. Other information for a given 
+    Some information in the output PVL file will consist of a compilation of information for all the
+    bands in the cube and will be listed in the "Common" object of the PVL. Other information for a given
     band will be placed in the "BandSet" object of the PVL. The user can select the check boxes to control
-    the information that will be output to the PVL file. If the POLYGON or USELABEL option is chosen, 
+    the information that will be output to the PVL file. If the POLYGON or USELABEL option is chosen,
     the image polygon is output in Well-Known Text format (WKT).
   </p>
-  <p>  
+  <p>
     The output file can be in PVL or CSV (Comma Separated Value) format. The "APPEND" option allows
-    the new generated information to be appended to the output file. This is especially useful for 
-    CSV file format. If the CSV format is chosen, only Camstats, Statistics and Geometry options 
-    are allowed. Isis Label, Original Label and Polygon options are disabled for CSV format. 
+    the new generated information to be appended to the output file. This is especially useful for
+    CSV file format. If the CSV format is chosen, only Camstats, Statistics and Geometry options
+    are allowed. Isis Label, Original Label and Polygon options are disabled for CSV format.
   </p>
-  <p> 
+  <p>
     The following is an example of caminfo output generated when in PVL format:
   </p>
-<pre> 
-Object = Caminfo
-  Object = Parameters
-    Program     = caminfo
-    IsisVersion = "3.4.2.0 alpha | 2012-08-28"
-    RunDate     = 2012-11-23T18:21:42
-    IsisId      = MeSSEnGeR/MDIS-WAC/1/0250507146:955000
-    From        = EW0250507146G.lev1.cub
-    Lines       = 512
-    Samples     = 512
-    Bands       = 1
-  End_Object
+<pre>
+  Object = Caminfo
+    Object = Parameters
+      Program     = caminfo
+      IsisVersion = "3.5.2.0 beta | 2017-11-04"
+      RunDate     = 2018-03-30T20:41:47
+      IsisId      = MeSSEnGeR/MDIS-WAC/1/0131773041:923000
+      From        = EW0131773041G.cal.cub
```

Comment:
Why did you choose to change the example?

Reply:
I told her to do that.  She was having difficulty finding the original cube that was used in the example, so I pointed her to another Messenger cube.